### DISCUSSION TITLE
docs/computer_science/projects/sokoban

### DISCUSSION THREAD

**Discussion by giscus** [LifeCheckpoint/LifeCheckpoint.github.io#2] [Announcements]
# docs/%E4%B8%8D%E5%8F%AF%E4%BB%A5%E5%81%9A%E6%9C%BA%E6%A2%B0%E9%99%8D%E7%A5%9E%E6%B4%BE/%E5%AE%8C%E6%95%B4%E7%9A%84%E9%A1%B9%E7%9B%AE%E4%B8%8D%E5%89%A9%E4%B8%A4%E4%B8%AA%E4%BA%86%E5%91%A2/sokoban

我想了想，下次还是不要把期末作业当成大项目搞了

https://lifecheckpoint.github.io/docs/%E4%B8%8D%E5%8F%AF%E4%BB%A5%E5%81%9A%E6%9C%BA%E6%A2%B0%E9%99%8D%E7%A5%9E%E6%B4%BE/%E5%AE%8C%E6%95%B4%E7%9A%84%E9%A1%B9%E7%9B%AE%E4%B8%8D%E5%89%A9%E4%B8%A4%E4%B8%AA%E4%BA%86%E5%91%A2/sokoban

**Comment by SweetDumplingPorridge** [comment]
是大佬，awsl

**Reply by LifeCheckpoint** [reply]
ヾ(•ω•`)o

**Comment by HuaQiting** [comment]
大佬交个朋友吧qaq

**Reply by LifeCheckpoint** [reply]
欢迎共同交流讨论（＾∀＾●）ﾉｼ

**Comment by tyexny** [comment]
大佬交个朋友吧qwq

**Comment by SweetDumplingPorridge** [comment]
话说大佬写这个的时候，一开始看的教程是什么啊……我现在也想要用libgdx写游戏。但中文网上连一个靠谱的教程都找不到……

**Reply by LifeCheckpoint** [reply]
我看的是官网英文教程哦，如果有什么其他不太清楚的，可能就得到网上去搜索有没有相似的问题，或者问问AI，看看源码...
